### PR TITLE
Fix #3532: Enable ownership tests for Team entity

### LINKED ISSUE #3532: Enable ownership tests for Team entity
**Affected module**
Testing

**Describe the bug**
Currently, in TeamResourceTest the flag `supportsOwner` is set to false. This should be removed and test failures should be fixed.

**To Reproduce**
Remove setting `supportsOwner` to false and run TeamResourceTest. Multiple tests fail.

### PR DESCRIPTION
**Pull request by harshach**
### Describe your changes :
See #3532 

#
### Type of change :

- [x] Bug fix

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [ ] I have performed a self-review of my own. 
- [ ] I have tagged my reviewers below.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: catalog-rest-service/src/test/java/org/openmetadata/catalog/resources/teams/TeamResourceTest.java [hunk 1]
```diff
@@ -76,7 +76,7 @@
 
   public TeamResourceTest() {
     super(Entity.TEAM, Team.class, TeamList.class, "teams", TeamResource.FIELDS);
-    this.supportsOwner = false; // TODO fix the test failures after removing this
+    this.supportsOwner = true;
```
Comment (sureshms) [inline catalog-rest-service/src/test/java/org/openmetadata/catalog/resources/teams/TeamResourceTest.java:79]: You don’t need to set the flag true. It is automatically detected.

**Review by vivekratnavel — APPROVED** [review]

**Comment by vivekratnavel** [comment]
@harshach Can you please fix checkstyle?

**Comment by harshach** [comment]
@vivekratnavel checkstyle is passing right https://github.com/open-metadata/OpenMetadata/runs/5638535839?check_suite_focus=true

**Comment by vivekratnavel** [comment]
You are right. For some reason, the summary marks it as failed. And it displays the statistics of unit test failure instead of checkstyle
Screen Shot 2022-03-21 at 10 52 05 PM

**Pull request merged**